atom/browser/api/atom_api_window.cc

```diff
@@ -659,6 +683,12 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("getMaximumSize", &Window::GetMaximumSize)
       .SetMethod("setResizable", &Window::SetResizable)
       .SetMethod("isResizable", &Window::IsResizable)
+      .SetMethod("setMovable", &Window::SetMovable)
+      .SetMethod("isMovable", &Window::IsMovable)
+      .SetMethod("setMinimizable", &Window::SetMinimizable)
+      .SetMethod("isMinimizable", &Window::IsMinimizable)
+      .SetMethod("setClosable", &Window::SetClosable)
+      .SetMethod("isClosable", &Window::IsClosable)
```

Comment:
I'm a little unsure whether `closable` or `closeable` is the right word here.

From a little googling, it looks like Java uses [closeable](https://docs.oracle.com/javase/1.5.0/docs/api/java/io/Closeable.html)